config/rbac/role.yaml

```diff
@@ -28,6 +28,20 @@ rules:
   - patch
   - update
   - watch
+- apiGroups:
+  - bootstrap.cluster.x-k8s.io
+  - exp.infrastructure.cluster.x-k8s.io
```

Comment:
should this be added to the apiGroups above instead?

Reply:
These should be automatically generated

Reply:
why are `bootstrap.cluster.x-k8s.io` and  `infrastructure.cluster.x-k8s.io` duplicated then?

Reply:
I haven't looked at the controller-tools in a while, but IIRC it's because there is two different `kubebuilder` directives which the code can't dedupe

Reply:
not allowing all resources for the experimental group can be an extra gate when / if needed. LGTM as per the existing `exp.cluster.x-k8s.io`.

also this ClusterRole file seems quite messy, indeed.